R/bias.R

```diff
@@ -247,71 +246,65 @@ bias_range <- function(range, lower, upper,
 #'
 #' Bias can assume values between
 #' -1 and 1 and is 0 ideally.
-#' @param predictions vector of length corresponding to the number of quantiles
-#' that holds predictions
-#' @param quantiles vector of corresponding size with the quantiles for which
-#' predictions were made
-#' @inheritParams bias_range
+#' @param lower vector of length corresponding to the number of central
+#' prediction intervals that holds predictions for the lower bounds of a
+#' prediction interval
+#' @param upper vector of length corresponding to the number of central
+#' prediction intervals that holds predictions for the upper bounds of a
+#' prediction interval
+#' @param range vector of corresponding size with information about the width
```

Comment:
Unrelated to the work here but noting for future reference as going through the code:

Should be stated here that these are given as percentages in [0,100]. Or even better perhaps they should be given as decimals in [0, 1] as that's what we're doing with quantiles.

Reply:
agree shall we track these suggestions in #309?

Reply:
Sounds good to me, yes.

Reply:
> Or even better perhaps they should be given as decimals in [0, 1] as that's what we're doing with quantiles

This would be a breaking change so might want to take some care here.